src/test/codegen/repr-transparent-x86_64.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// only-x86_64
```

Comment:
I'm not sure but this might need more restrictions, since not all x86_64 targets use the SysV ABI. Or alternatively, make the function below `extern "sysv64"` as in the other codegen test.

Reply:
Right, I forgot to add the other only-XXX line that I meant to add, but I think forcing sysv ABI is even better here, so I'll go with that. Thanks for the suggestion